modules/cnf-managing-device-interrupt-processing-for-guaranteed-pod-isolated-cpus.adoc

```diff
@@ -0,0 +1,58 @@
+// CNF-802 Infrastructure-provided interrupt processing for guaranteed pod CPUs
+// Module included in the following assemblies:
+//
+// *cnf-performance-addon-operator-for-low-latency-nodes.adoc
+
+[id="managing-device-interrupt-processing-for-guaranteed-pod-isolated-cpus_{context}"]
+= Managing device interrupt processing for guaranteed pod isolated CPUs
+
+The Performance Addon Operator manages host CPUs by dividing them into reserved CPUs for cluster housekeeping duties, and isolated CPUs for workloads. CPUs that are used for real-time workloads are set as isolated. Being able to configure device interrupt processing means that administrators can balance the number of reserved CPUs against the number of guaranteed pod CPUs for low-latency processing. In the performance profile, `GloballyDisableIrqLoadBalancing` is used to manage whether device interrupts are processed or not.
+
+To achieve real-time latency for workloads, some (but not all) pods require the CPUs they are running on to not process device interrupts.
+
+For certain workloads the reserved CPUs are not always sufficient for dealing with device interrupts, and for this reason, device interrupts are not globally disabled on the isolated CPUs. By default, PAO does not disable device interrupts on isolated CPUs. A pod annotation, `irq-load-balancing.crio.io`, is used to define whether device interrupts are processed or not.
+
+When configured, CRI-O disables device interrupts only as long as the pod is running.
+
+[id="configuring-global-device-interrupts-handling-for-isolated-cpus_{context}"]
+== Configuring global device interrupts handling for isolated CPUs in Performance Addon Operator
+
+To configure Performance Addon Operator to disable global device interrupts for the isolated CPU set, set the `GloballyDisableIrqLoadBalancing` field in the performance profile to `true`. A performance profile snippet illustrates this setting:
+
+[source,yaml]
+----
+apiVersion: performance.openshift.io/v2
+kind: PerformanceProfile
+metadata:
+  name: manual
+spec:
+  GloballyDisableIrqLoadBalancing:
```

Comment:
I think in yaml examples first letter should be in lowercase: _globallyDisableIrqLoadBalancing_

Reply:
Are you sure? I see camel case in https://github.com/openshift-kni/performance-addon-operators/pull/424

Reply:
in CRD we have first letter in lowercase: 

https://github.com/openshift-kni/performance-addon-operators/blob/716eefc168c55d4aab176de4f1a4549007a215e2/deploy/olm-catalog/performance-addon-operator/4.7.0/performance.openshift.io_performanceprofiles_crd.yaml#L311

so as I know - in yaml files (only) it should also be the same, otherwise it will not be applied.
@marcel-apf  can you please confirm?

Reply:
@dshchedr I've made the change as you suggested acorss all instances `GloballyDisableIrqLoadBalancing` > `globallyDisableIrqLoadBalancing`. @marcel-apf please chime in if this is incorrect.